docs/pages/database-access/reference/configuration.mdx

```diff
@@ -24,7 +24,14 @@ The following Proxy service configuration is relevant for Database Access:
   Do not set `--insecure-no-tls` for the proxy Teleport instances as a parameter.  If you are terminating TLS at a
   Application Load Balancer (ALB) or other service that may require enabling a backend protocol of HTTPS for the target address.  
 </Admonition>
-
+<Admonition
```

Comment:
Two admonitions back to back like this is not ideal. @ptgott usually has good suggestions to break them up.

Reply:
Thanks—@stevenGravy what about turning the second Admonition into part of the comments in the `proxy_service` config snippet? We could add this before we start listing `_public_addr` fields. We could add one or two line breaks between the `_listen_addr` fields and the `_public_addr` fields so this warning is more prominent. 

Another option is to remove both Admonitions and write a proper introductory section before the configuration snippet. This would explain what the role of the Proxy Service is for Database Access and, broadly, how to use the `_listen_addr` and `_public_addr` fields, including a restructured version of the two Admonitions as part of the text.